src/test/kotlin/org/rust/ide/inspections/RsAttrWithoutParenthesesInspectionTest.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.inspections
+
+class RsAttrWithoutParenthesesInspectionTest : RsInspectionsTestBase(RsAttrWithoutParenthesesInspection::class) {
```

Comment:
Let's add at least three additional tests:
- with custom attribute with child attribute from `attributesWithParen` list, something like
```rust
#[custom_attr(repr)]
struct Foo(i32);
```
- something with `cfg_attr` where, ideally, we should detect the same error but don't do it yet
```rust
#[cfg_attr(unix, repr)]
struct Foo(u8);
```
- case with an inner attribute like `#![warn]`